### DISCUSSION TITLE
Adopt a pattern! Adopt a page!

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#1962] [Forum]
Over the past month, we've been discussing pattern ambassadors and page maintainers. Let me know if these things should be separate Discussions. Relevant meetings: [February 5th](https://github.com/freesewing/freesewing/discussions/1699) and [February 19th](https://github.com/freesewing/freesewing/discussions/1833).
Also relevant: @bobgeorgethe3rd's [bubble idea](https://github.com/freesewing/freesewing/discussions/1781).

Looking after a pattern or page is, for most people, easier to do than coding a pattern or fixing a bug. An 'entry-level position', if you will.
The idea is that the contributor's name and [icon](https://github.com/freesewing/freesewing/discussions/1781) will be shown on the pattern or page they're maintaining. Like a badge of honor 😎

## Adopt a pattern
Tasks of a pattern maintainer:
- Make sure the documentation makes sense and keep it up-to-date.
- Make sure drafting it works.
- Check if a new version doesn't break the pattern, if it does: contact a developer if they're not a developer themself and get a fix or warning on the page.

## Adopt a page
Tasks of a page maintainer:
- Make sure the page has content. Check for spelling mistakes or things that are missing.
- Make sure the page doesn't get neglected.
- Be open to update/add things if needed.

### The social aspect
It needs to be clear that this position exists. We can put it on the site, something like, (credits to @bobgeorgethe3rd:) "Love this pattern? It needs an ambassador to help take care of it" and we can also make an announcement on the Discord. Maybe even a blog post?

### The technical aspect
The idea is that a contributor's name and icon are on the page they maintain and other pages have a statement saying it could use a maintainer. Relevant idea mentioned before: that a pattern's developer(s), designer(s), documentation writer(s) and other contributors are shown on the page as well.

### It needs to be maintained
We need to make sure that if a person drops out, the site will be updated. There should also be a list of patterns and pages, and their maintainers. (We can make a Discussion for this when the time comes.)  

Feel free to use this Discussion to brainstorm. Also, I feel like this is gonna take some time to implement, so no rush.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
